### ISSUE TITLE
Schedule : resizable attribute not being honored

### ISSUE THREAD

**Issue by tarwork** [primefaces/primefaces#6117]
**Describe the defect**
The resizable attribute for p:schedule is not being honored in primefaces-8.0.jar

**Reproducer**
use the community download version of PF 8.0 from https://www.primefaces.org/downloads 

**Environment:**
 - PF Version: _8.0_
 - JSF + version: Mojarra 2.2.8
 - Affected browsers: Chrome

**To Reproduce**

The resizable attribute for p:schedule is not being honored in primefaces-8.0.jar 

 <p:schedule id="myschedule" resizable="false" ... > .... </p:schedule> will still allow me to resize individual entry items.

**Expected behavior**
The resizable attribute for p:schedule should be honored in primefaces-8.0.jar

**Comment by melloware** [comment]
There is no `resize` attribute on schedule only `resizable`?

See: https://primefaces.github.io/primefaces/8_0/#/components/schedule?id=schedule

**Comment by tarwork** [comment]
Thank you. original post has been updated. I mean resizable.

**Comment by melloware** [comment]
Hmm resisable is being set in the renderer here:
https://github.com/primefaces/primefaces/blob/cd4fbdf1d9d4ae054da19b9a84001d0c34d142eb/src/main/java/org/primefaces/component/schedule/ScheduleRenderer.java#L185

The docs for `eventDurationEditable` are here: https://fullcalendar.io/docs/eventDurationEditable

**Comment by melloware** [comment]
@blutorange I think this might also be fixed by your PR: https://github.com/primefaces/primefaces/pull/6107

I just tested this against 8.0 and currently resizable="false" or "true" has no effect and the events are not resizable.

**Comment by tarwork** [comment]
it does look like if I set each event to not be editable with this : setEditable(false) before I put each event in the EventModel then yes the resize is disabled. However, the draggable is also disabled. Unfortunately I need draggable for my user requirements otherwise that could have been a short-term work around.

thank you for confirming.

**Comment by blutorange** [comment]
@melloware Yeah, that's exactly what I fixed. No matter what was set on the schedule, it was always overwritten by the individual events (editable=true/false). Now it defaults to null, which doesn't overwrite the default.

@tarwork You can still work around this: Use DefaultScheduleEvent#dynamicProperty (https://github.com/primefaces/primefaces/blob/master/src/main/java/org/primefaces/model/DefaultScheduleEvent.java#L326) and set the properties `startEditable` (=draggable) and `durationEditable` (=resizable) to true and false. 

You can see all available properties here: https://fullcalendar.io/docs/event-object

**Comment by melloware** [comment]
Perfect I will link your PR to this ticket as well.

**Comment by tuerker** [comment]
@blutorange: the attributes draggable and scrollTime have the same problem. would be great if these issues are fixed, too. many thanks.

**Comment by tuerker** [comment]
Same for the attribute displayEventEnd.

**Comment by blutorange** [comment]
* `scrollTime` What do you mean? This is a global attribute only and can not be set for individual events. I just tried with PF9 Snapshot and my PR, and there at least it works. Keep in mind though that `scrollTime` only set the initial scrolling position of the calendar view - users can still scroll up.
* `draggable` is what I fixed when I updated full calendar to 5.1.0
* `displayEventEnd`: With the PF 9 snapshot and full calendar at 5.1.0 it seems to be working:

`displayEventEnd=false`

![image](https://user-images.githubusercontent.com/2456413/87072854-60f49e80-c214-11ea-9db7-3ac5617dfebd.png)

`displayEventEnd=true`

![image](https://user-images.githubusercontent.com/2456413/87072894-6f42ba80-c214-11ea-982e-bac9e7e8f81c.png)

**Comment by tuerker** [comment]
@blutorange: Thanks for your work. I will be happy to checkout your fixed version.

**Comment by melloware** [comment]
@tuerker You can try it right now by using...
```xml
 <dependency>
     <groupId>com.github.primefaces</groupId>
     <artifactId>primefaces</artifactId>
     <version>master-SNAPSHOT</version>
 </dependency>

 <repositories>
 	<repository>
 	    <id>jitpack.io</id>
 	    <url>https://jitpack.io</url>
 	</repository>
 </repositories>
```

**Comment by sasynkamil** [comment]
Unfortunatelly this bugfix is urgent for us. I haven't found any planned date for PF 9 release. But based on the previous releases, Am I right PF 9 will be released around March or April 2021?

**Comment by melloware** [comment]
@sasynkamil PrimeTek had talked about releasing PF 9.0 in October/November timeframe so it may be sooner than next Spring.

**Comment by blutorange** [comment]
@sasynkamil If you have time it would also be great if you could try out the current snapshot version (https://github.com/primefaces/primefaces/issues/6117#issuecomment-656624814) and see if it solves your problem.

**Comment by sasynkamil** [comment]
@blutorange I gave it try, but no luck with switching from PF 8.0  to the latest snapshot. The project we are maintaining contains ohter "deprecated" stuff like  p:editor > p:textTeditor, p:layout > pe:layout and some others. Even if I switched these deprecated components to theirs successtors, UI is broken and I'm getting JS exceptions. I have no time now and I don't promiss, but later I will try to simulate this issue on primefaces test project and then verify bug fix with the snapshot. Thank you for your work.
